frontend/package.json

```diff
@@ -19,6 +19,7 @@
     "jabber-shared": "file:../shared/build",
     "react": "17.0.2",
     "react-dom": "17.0.2",
+    "react-hook-form": "7.12.1",
```

Comment:
Add @types package as well (If applicable)

Reply:
there are no types for this library 🙂

there are out the box